tests/test_transform.py

```diff
@@ -93,3 +98,25 @@ def test_snake_to_dromedary(string: str, expected: str) -> None:
 def test_dromedary_to_snake(string: str, expected: str) -> None:
     result = dromedary_to_snake(string)
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    ("string", "expected"),
+    [
+        ("", ""),
+        ("word", "word"),
+        ("already-kebab", "already-kebab"),
+        ("ABWordCDEWordFG", "ab-word-cde-word-fg"),
```

Comment:
Maybe add a case for `Such-AWeird-EdgeCase`?

Reply:
done [c2d6669](https://github.com/robert-koch-institut/mex-common/pull/48/commits/c2d6669d09cf6aa3b5e0d46b479cd3de7d7d5cae)